src/pages/docs/deployments/bulk-deployment-creation.md

```diff
@@ -0,0 +1,46 @@
+---
+layout: src/layouts/Default.astro
+pubDate: 2023-09-13
+modDate: 2023-09-13
+title: Bulk deployment creation
+description: Octopus Deploy can use bulk deployment creation when deploying to multiple environments or tenants
+navOrder: 160
+---
+
+:::div{.hint}
+Bulk deployment creation will be available from Octopus Deploy **2023.4**
+:::
+
+When deploying a release to multiple environments or tenants through the Octopus Portal, the bulk deployment creation feature will immediately take you to a new server task which will create a deployment for each environment or tenant you are deploying to.
+
+In the bulk deployment creation server task, you can see the progress of deployments being created and any errors during creation.
```

Comment:
Can consolidate this now with above:

There you will find each requested deployment recorded as an indvidual task, which will display progress and any errors.